src/model/types.ts

```diff
@@ -3,12 +3,25 @@ import { ApiResponse as ApiResponse6 } from 'es6';
 import { ApiResponse as ApiResponse7 } from 'es7';
 import { ApiResponse as ApiResponseOS } from '@opensearch-project/opensearch';
 import { MigrationPlan } from '../executor/plan/MigrationPlan';
+import Elasticsearch6Client from '../utils/es/6/Elasticsearch6Client';
```

Comment:
There seems to be an unnecessary import. Can you delete it?

Reply:
you're correct, will remove it